benefits/eligibility/views.py

```diff
@@ -217,16 +217,17 @@ def unverified(request):
     # tel: link to agency phone number
     agency = session.agency(request)
     buttons = viewmodels.Button.agency_contact_links(agency)
+    buttons.append(viewmodels.Button.home(request, _("enrollment.buttons.retry")))
```

Comment:
This wasn't the focus of this ticket, but it revealed the issue nonetheless...


Now that the Unverified page has a Back button, there is a possible flow where the user is already signed in with Login.gov and they land on the Start page. We don't make them login again, but show a "Continue" button instead, which needs to be styled to right-align like the Login.gov button does:

![image](https://user-images.githubusercontent.com/1783439/165805047-25c0a6a6-8e22-4422-8e7b-41a3ee5225ac.png)

Reply:
Made new issue: https://github.com/cal-itp/benefits/issues/511